keps/sig-api-machinery/20190425-crd-conversion-webhook.md

```diff
@@ -736,8 +735,23 @@ Consider including folks that also work outside the SIG or subproject.
 
 ## Graduation Criteria
 
+v1beta1:
 
 
+- Ensure the scenarios from (https://github.com/kubernetes/kubernetes/issues/64136) are tested:
+  - Ensure what is persisted in etcd matches the storage version
+  - Set up a CRD, persist some data, change the storage version, and ensure the previously persisted data is readable
+  - Ensure discovery docs track a CRD through creation, version addition, version removal, and deletion
+  - Ensure `spec.served` is respected
```

Comment:
a few more test scenarios we need covered for beta:
* error-case handling when calling conversion results in an error during {get, list, create, update, patch, delete, deletecollection} calls to the primary resource
* error-case handling when calling conversion results in an error during {get, update, patch} calls to the status subresource
* error-case handling when calling conversion results in an error during {get, update, patch} calls to the scale subresource

Reply:
Thanks. Added.